libs/src/services/creatorNode/index.js

```diff
@@ -369,13 +370,13 @@ class CreatorNode {
       // Should have a body structure of:
       //   { transcodedTrackCID, transcodedTrackUUID, track_segments, source_file }
       if (status && status === 'DONE') return resp
-      if (status && status === 'FAILED') await this._handleErrorHelper(new Error(`Transcode failed: uuid=${uuid}, error=${resp}`), `/track_content_async`, uuid)
+      if (status && status === 'FAILED') await this._handleErrorHelper(new Error(`Transcode failed: uuid=${uuid}, error=${resp}`), route, uuid)
 
-      // Check the transcode status every 5s
+      // Check the transcode status every 3s
```

Comment:
tbh since this is defined as a var, idk if you needa write 3s here (if the var changes, someone may forget this comment)

Reply:
I can remove this. this line is self explanatory enough